Remotion/ObjectBinding/Web/UI/Controls/BocTreeNode.cs

```diff
@@ -163,47 +171,52 @@ private void EnsureBusinessObject()
     }
     else
     {
-      IBusinessObjectReferenceProperty property = Property;
+      IBusinessObjectReferenceProperty? property = Property;
       string businessObjectID = ItemID;
       BusinessObject = ((IBusinessObjectClassWithIdentity) property.ReferenceClass).GetObject (businessObjectID);
```

Comment:
Why does this compile?

Reply:
Compiler knows property is not null. I removed the `?`.